core/src/main/java/com/datastax/oss/driver/internal/core/context/StartupOptionsBuilder.java

```diff
@@ -142,4 +154,21 @@ protected String getDriverName() {
   protected String getDriverVersion() {
     return Session.OSS_DRIVER_COORDINATES.getVersion().toString();
   }
+
+  private Optional<String> driverBaggage() {
+    ImmutableMap.Builder<String, Object> builder = new ImmutableMap.Builder<>();
+    for (Map.Entry<String, LoadBalancingPolicy> entry :
+        context.getLoadBalancingPolicies().entrySet()) {
+      Map<String, ?> config = entry.getValue().getStartupConfiguration();
```

Comment:
We should offer a configuration to disable getStartupConfiguration that matches the current behavior, for users with tons of profiles or custom policies that don't support it properly yet

Reply:
Yeah, fine with me